app/uk/gov/hmrc/gform/submission/destinations/DestinationSubmissionInfo.scala

```diff
@@ -17,8 +17,13 @@
 package uk.gov.hmrc.gform.submission.destinations
 
 import uk.gov.hmrc.gform.sharedmodel.form.FormId
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.FormTemplateId
 import uk.gov.hmrc.gform.submission.Submission
 
-case class DestinationSubmissionInfo(customerId: String, submission: Submission) {
+case class DestinationSubmissionInfo(
+  customerId: String,
+  submission: Submission,
+  formTemplateId: FormTemplateId
```

Comment:
submission.dmsMetaData.formTemplateId 
is that the same formTemplateId as a new `formTemplateId` argument?

Reply:
That's the same formTemplateId to pass into DestinationSubmitter.

Reply:
FormTemplateId exists in DmsMetaData. I'll remove it.